src/org/loklak/harvester/strategy/KaizenHarvester.java

```diff
@@ -61,7 +62,10 @@ public KaizenHarvester() {
 
         random = new Random();
 
-        twitter = TwitterAPI.getAppTwitterFactory().getInstance();
+        TwitterFactory twitterFactory = TwitterAPI.getAppTwitterFactory();
+
+        if (twitterFactory != null)
+            twitter = twitterFactory.getInstance();
 
         if (twitter == null)
```

Comment:
Though both the cases will result in same outcomes, the first condition is to check if the `twitterFactory` is `null`. While the second is to give out a log if `twitter` is `null`, and hence is not the * else * part of the first.